src/script.js

```diff
@@ -40,17 +15,67 @@ createTheModal = () => {
       globalModal.style.display = "none";
     }
   }
+  console.log('window', window);
 }
 
-let modalInformation = document.getElementById("modalInformation")
-console.log('modalInformation', modalInformation);
+globalInfo = () => {
 
+  modal.style.display = "block";
+  //Create Basic information about school
```

Comment:
If this information is all static (won't change based on some http call), I'd put all of this in the html. Dynamically creating static content is not really what we use js for